ci/libs/utils.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+
+. ci/libs/variables.sh
+
+cleanup(){
+	rm -rf $TMP_DIR/vol*
+	rm -rf $MNT_DIR/logs
+    docker stop $(docker ps -aq)
+    docker rm $(docker ps -aq) 
```

Comment:
same with this do we really want to clean all docker containers or the once we started

Reply:
The same method is used to cleanup the containers before starting the test